core/store/migrations/0034_external_job_id.go

```diff
@@ -0,0 +1,33 @@
+package migrations
+
+import "gorm.io/gorm"
+
+const (
+	up34 = `
+               CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
I'm confused about our approach here. I see in some cases we are deliberately generating it on the client side, but also we autogenerate it if null. Seems ambiguous.

I'd prefer to pick one or the other if possible?

In addition, there are problems with using `CREATE EXTENSION`. Many nops run their chainlink databases with non-superuser users who do not have permissions to do this.

Reply:
The autogenerate if null is to handle existing records in the DB i.e. for FMv2/OCR jobs. Hm if we can't rely on create extension being available I'm not sure there's a way to make it not null - any ideas?

Reply:
Spoke offline